src/screen/unix.rs

```diff
@@ -256,13 +254,13 @@ impl Terminal {
     }
 
     fn translate_bytes(bytes: Vec<u8>) -> Vec<Key> {
-        if bytes == [27, '[' as u8, 'A' as u8] { return vec![Up] };
-        if bytes == [27, '[' as u8, 'B' as u8] { return vec![Down] };
-        if bytes == [27, 'O' as u8, 'A' as u8] { return vec![Up] };
-        if bytes == [27, 'O' as u8, 'B' as u8] { return vec![Down] };
+        if bytes == [27, b'[' as u8, b'A' as u8] { return vec![Up] };
+        if bytes == [27, b'[' as u8, b'B' as u8] { return vec![Down] };
+        if bytes == [27, b'O' as u8, b'A' as u8] { return vec![Up] };
+        if bytes == [27, b'O' as u8, b'B' as u8] { return vec![Down] };
```

Comment:
The `as u8` cast is actually unnecessary now, since the single quotes result in a [byte literal](https://doc.rust-lang.org/reference.html#byte-literals) rather than a byte string.